src/pages/lessons/[slug].tsx

```diff
@@ -416,19 +431,17 @@ const Lesson: FunctionComponent<LessonProps> = ({initialLesson}) => {
                         send('PAUSE')
                       }}
                       onProgress={({...progress}) => {
-                        onProgress(progress).then((lessonView: any) => {
+                        onProgress(progress, lesson).then((lessonView: any) => {
```

Comment:
but, unlike `onEnded` this one has a reference to the correct `lesson` instance 😭